packages/specification/schemas/fragments/groupIdentifier.yaml

```diff
@@ -0,0 +1,17 @@
+type: object
+required: [value]
+description: Prescription ID to facilitate accessing a prescription in the event that a barcode scanner was inoperable.
```

Comment:
Worth highlighting that we would want a client to use this to generate a prescription barcode.

Reply:
Added to description to reflect this.